code/game/objects/items/weapons/his_grace.dm

```diff
@@ -158,6 +163,8 @@
 	else
 		bloodthirst = HIS_GRACE_CONSUME_OWNER
 	update_stats()
+	if(LAZYLEN(contents) >= victims_needed)
+		ascend()
```

Comment:
This should only happen if the user actually used the toolbox to consume a target, rather than if the toolbox consumed someone, or its master on its own.